example/partials/grid/_editor.yaml

```diff
@@ -0,0 +1,17 @@
+label: Grid
```

Comment:
Shouldn't this be in the front matter for the html?

Reply:
Yeah, it should be. But I was copying the existing convention in the example repo.

Reply:
Will keep this as-is until we move the other examples in this repo.